#src/scss/blocks/hero.scss

```
@@ -0,0 +1,22 @@
+@import "../lib/variables.scss";
+.hero-row {
+    position: relative;
+}
+.hero__header {
+    font-family: $fontHeading;
+    font-size: 64px;
+    font-weight: bold;
+    color: $colorDark;
+    margin-bottom: 74px;
+    line-height: 1.3em;
+    position: relative;
+}
+.hero__header::after {
+    content: "";
+    background: url("../../img/plus.svg") no-repeat;
+    width: 20px;
+    height: 20px;
+    position: absolute;
+    left: 445px;
```

better position it starting from right, since width may be different on different screens, say for example this block will be 300 on smaller screen, then this pseudoelement will be far away
Instead, if you `right` as shown below, it will always be X pixels away from the right (end of the block)
```suggestion
    right: -5px; // or whatever -X px
```